src/PowerSystems2PRAS.jl

```diff
@@ -337,7 +336,11 @@ function process_generators(
             end
         end
 
-        λ_gen[idx, :], μ_gen[idx, :] = get_outage_time_series_data(g, s2p_meta)
+        λ_gen[idx, :], μ_gen[idx, :] = get_outage_time_series_data(
+            g,
+            s2p_meta,
+            get_add_default_transition_probabilities(component_to_formulation[g]),
```

Comment:
I spent some time trying to get this working, but I didn't spend _too_ much time. This seemed to fail in a few different cases:
- some component formulations don't have defaults
- this is incompatible with the lumping, since lumped generators are not added to the formulation